operator/pkg/helmreconciler/apply.go

```diff
@@ -89,6 +90,20 @@ func (h *HelmReconciler) ApplyManifest(manifest name.Manifest) (object.K8sObject
 		scope.Infof("Generated manifest objects are the same as cached for component %s.", cname)
 	}
 
+	// check minor version only
+	serverSideApply := false
+	if h.restConfig != nil {
+		k8sVer, err := k8sversion.GetKubernetesVersion(h.restConfig)
```

Comment:
Can we do this check outside of ApplyManifest? its called multiple times right?

Reply:
good point, moved to outside of the loop